Clarinet.toml

```diff
@@ -1,47 +1,32 @@
 [project]
 name = "executor-dao"
 requirements = []
-costs_version = 2
+costs_version = 1
```

Comment:
Why was it changed to `1`?

Reply:
I was testing what the difference is. Will revert to `cost_version = 2`.